src/Homeworks/Homework4_Task2/Parse Tree/UtilsPT/NodeOperand.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) AstrArina. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace ParseTree;
+
+public class OperandNode(double number) : IParseTreeNode{
```

Comment:
```suggestion
public class OperandNode(double number) : IParseTreeNode
{
```
И об этом тоже StyleCop должен был подсказать